src/bionics_ui.cpp

```diff
@@ -510,175 +511,135 @@ void player::power_bionics()
             if( cursor > 0 ) {
                 cursor--;
             } else {
-                cursor = current_bionic_list->size() - 1;
+                cursor = current_bionic_list.size() - 1;
             }
             if( scroll_position > 0 && cursor - scroll_position < half_list_view_location ) {
                 scroll_position--;
             }
             if( scroll_position < max_scroll_position &&
                 cursor - scroll_position > LIST_HEIGHT - half_list_view_location ) {
                 scroll_position =
-                    std::max( std::min<int>( current_bionic_list->size() - LIST_HEIGHT,
+                    std::max( std::min<int>( current_bionic_list.size() - LIST_HEIGHT,
                                              cursor - half_list_view_location ), 0 );
             }
-        } else if( menu_mode == REASSIGNING ) {
-            menu_mode = ACTIVATING;
-
-            if( action == "CONFIRM" && !current_bionic_list->empty() ) {
-                auto &bio_list = tab_mode == TAB_ACTIVE ? active : passive;
-                tmp = bio_list[cursor];
-            } else {
-                tmp = bionic_by_invlet( ch );
-            }
-
-            if( tmp == nullptr ) {
-                // Selected an non-existing bionic (or Escape, or ...)
-                continue;
+        } else if( action == "REASSIGN" ) {
+            tmp = current_bionic_list[cursor];
+            while( true ) {
+                const int invlet = popup_getkey(
+                                       _( "%s\n\nEnter new letter. Press SPACE to clear a manually assigned letter, ESCAPE to cancel." ),
```

Comment:
```suggestion
                                       _( "%s\n\nEnter new letter.  Press SPACE to clear a manually assigned letter, ESCAPE to cancel." ),
```

Reply:
Why is this a thing?

Reply:
[Manual of style.](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md)
[Why?](https://github.com/CleverRaven/Cataclysm-DDA/pull/5414#issuecomment-31611671)

Reply:
Ok, thanks